src/targets/gpu/lowering.cpp

```diff
@@ -146,6 +149,7 @@ struct miopen_apply
         add_generic_op("logical_xor");
         add_generic_op("max");
         add_generic_op("min");
+        add_generic_op("mod");
```

Comment:
These can be removed as this will not be used. The fuse_pointwise will replace them before it gets here to lowering.

Reply:
OH didn't realize that. Thanks! I can remove

Reply:
done.